rules/library.bzl

```diff
@@ -242,11 +242,28 @@ def _canonicalize_swift_version(swift_version):
 def _xcframework_build_type(*, linkage, packaging):
     return (linkage, packaging)
 
+def _xcframework_slice_imports(path):
+    """
+    Returns a tuple of the headers, module map and swiftmodules for the given path in a .xcframework.
+    """
+    import_headers = native.glob(["%s/**/*.h" % path])
+    import_module_maps = native.glob(["%s/**/*.modulemap" % path])
+    import_swiftmodules = native.glob(["%s/**/*.swiftmodule/*.*" % path])
```

Comment:
Can you add `allow_empty = True` to these 3 globs to match https://github.com/bazel-ios/rules_ios/pull/518? Unfortunately we can't add a test case for this config since analysis tests can't transition on `incompatible_` flags.

Reply:
Updated!